.nancyignore

```diff
@@ -0,0 +1,24 @@
+# Fixed in https://github.com/beego/beego/releases/tag/v1.12.11 which is available in https://github.com/goharbor/harbor/releases/tag/v2.5.4. However, it appears that it's more likely that this PR (https://github.com/goharbor/harbor-operator/pull/970) will be merged first which also takes in beegov1.12.11.
```

Comment:
Filename has to be `.nancy-ignore` I think

Reply:
you're right, will update 👍